explorer/images/discourse.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="15" viewBox="0 0 14 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Now that the [style guide icons](https://www.figma.com/file/i4uZsYkKAML9ivi2TIo6ug/explore-dashboard?node-id=470%3A5119) are available, let's:

- replace all svgs with the latest 20x20 view,
- remove any "fill" attributes from the `paths`,
- change the value of the "fill" attribute on the `svg` element to "currentColor"

This will allow us to re-use the svg and control the colour of the svg, should we wish to in the future.